modules/core/src/v2/internal/tssUtils.ts

```diff
@@ -575,4 +588,22 @@ export class TssUtils extends MpcUtils {
       .send({ txRequestId })
       .result();
   }
+
+  /**
+   * Delete signature shares, get the tx request without them from the db and sign it to finally send it.
+   *
+   * Note : This can be performed in order to reach latest network conditions required on pending approval flow.
+   *
+   * @param {String} txRequestId - the txRequest Id to make the requests.
+   * @param {String} walletPassphrase - the wallet phassphrase to sign the tx request.
+   * @param {RequestTracer} reqId id tracer.
+   * @returns {Promise<any>}
+   */
+  async recreateTxRequest(txRequestId: string, walletPassphrase: string, reqId: RequestTracer): Promise<any> {
+    await this.deleteSignatureShares(txRequestId);
+    // after delete signatures shares get the tx without them
+    const txRequest = await this.getTxRequest(txRequestId);
+    const txRequestSigned = await this.signTxRequest(txRequest, walletPassphrase, reqId);
+    return this.sendTxRequest(txRequestSigned.txRequestId);
```

Comment:
change return type to `Promise<TxRequest>` and return `txRequestSigned`

we can't use `sendTxRequest` as the final call to approve the transaction.
we need to use the existing update pending approval call instead.

if we do this, we'll create a new pending approval entry.

once you make these changes, the `sendApproval` API call will take care of broadcasting the transaction

Reply:
let me know if now is right.